packages/slice-machine/lib/env/manifest.ts

```diff
@@ -123,13 +72,32 @@ function handleManifest(cwd: string): ManifestInfo {
 
     return pipe(
       Models.Manifest.decode(json),
-      fold(
+      fold<t.Errors, Models.Manifest, ManifestInfo>(
         // failure handler
-        () => {
-          throw new Error();
+        (errors) => {
+          const messages = formatValidationErrors(errors, {});
```

Comment:
Is this really necessary ?

You're already sending custom errors through so is this needed ?

Reply:
yes, some of the types don't provide custom messages